pandas_gbq/gbq.py

```diff
@@ -1114,15 +1109,21 @@ def to_gbq(
                 stacklevel=2,
             )
 
-    if if_exists not in ("fail", "replace", "append"):
-        raise ValueError("'{0}' is not valid for if_exists".format(if_exists))
-
     if "." not in destination_table:
         raise NotFoundException(
             "Invalid Table Name. Should be of the form 'datasetId.tableId' or "
             "'projectId.datasetId.tableId'"
         )
 
+    if if_exists not in ("fail", "replace", "append"):
```

Comment:
minor consideration:  Should we case coerce `if_exists` to lowercase when evaluating, or throw the value error unless they're exact match?  I'm fine with the strictness approach, but something to consider.

Reply:
That's a good point. I'll make a note to do that